vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -87,6 +87,70 @@ For reference, see the [Vizro CSS files](https://github.com/mckinsey/vizro/tree/
     [AssetsCSS]: ../../assets/user_guides/assets/css_change.png
 
 
+## Overwrite CSS properties for specific pages
+If you want to style specific pages only, you need to provide a valid `id` to the page you want to modify.
+This unique `id` enables you to use CSS selectors to target specific pages and their components for styling.
+
+An HTML `id` is considered valid if the following conditions are met:
+- Use lowercase letters and hyphens
+- Avoid using special characters or spaces
+- Provide a unique `id`
```

Comment:
```suggestion
To style components for only a specific page, use the page's `id` in CSS selectors. By default, this is the [same as the page `title`](pages.md), but such a value might not be a valid CSS identifier. A suitable `id` must be unique across all models in the dashboard and should contain only alphanumeric characters, hyphens (`-`) and underscores (`_`). In particular, note that spaces are _not_ allowed.
```

This guidance is general about `id` really applies everywhere of course, not just to page's. Feel free to move it now or in the future when we restructure the page as discussed earlier.

The only special part that applies specifically to page is just that the `id` is automatically set and hence more likely to be inappropriate for CSS identifiers due to having a space in it.